spec/requests/api/v1/sessions/destroy_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe 'DELETE api/v1/users/sign_out', type: :request do
+  let(:user) { create(:user) }
+
+  subject { delete destroy_user_session_path, headers: auth_headers, as: :json }
+
+  it 'returns a successful response' do
+    subject
+    expect(response).to be_successful
+  end
```

Comment:
Nice, what about adding another expectation to check that the session keys are empty for that user.
I implemented that by doing 
```
expect(user.reload.tokens).to be_empty
```
a month ago, maybe it doesn't work with your current devise version, but you can give it a try